package/com.unity.formats.usd/Documentation~/USD Import.md

```diff
@@ -0,0 +1,53 @@
+# Importing USD Files
+
+![](Images/usd-import-options.PNG)
+
+To import a USD file into Unity, use the USD context menu, and select one of the 'Import as ...' options as required. Supported USD file formats are:
+
+-   .USD
+-   .USDA
+-   .USDC
+-   .USDZ
+
+## Importing Options
+
+-   **Import As GameObjects**: Imports selected USD file as GameObjects directly into the Scene Hierarchy.
+
+-   **Import as Prefab**: Imports selected USD file as a Prefab into the project folder.
+
+-   **Import as Timeline Clip**: Imports selected USD file as a Prefab containing empty Materials and the Timeline Clip. If animation is recorded in the Timeline Clip, that can be used to animate other applicable Unity Objects.
+
+## Importing Materials
+
+To import materials from USD, firstly import the USD file using the USD menu. Then, to get the materials, change the 'Import Settings' > 'Materials' in the Inspector to 'Import Preview Surface'. Finally re-import the USD file.
+
+## Importing Payloads
+
+Payloads are not loaded by default. If you are using payloads, in order to see anything in the scene,  firstly import the USD file using the USD menu. Then, to get the payloads, change the 'Import Settings' > 'Payload Policy' in the Inspector to 'Load All''. Finally re-import the USD file.
+
+## Primitive Sanitization on Import
+
+In order to fit Unity formats, during import we perform the following sanitization steps as required:
+
+### Cameras and Xforms
+
+Convert basis:
+-   Converting from authored orientation to Unity's left handed orientation. The advanced import settings allow different methods of converting basis:
+    -   Fast With Negative Scale: 
+    -   SlowAndSafe (default)
+    -   SlowAndSafeAsFBX
+    -   None
+
+### Meshes
+
+Convert basis:
+- As above, meshes also go through a basis conversion as required.
+
+Triangulate meshes:
+- Converting from USD-supported mixed face meshes to a triangular mesh.
+
+Change vertex winding order: 
+- Unity uses a clockwise mesh winding order, where other DCCs may use anti-clockwise.
```

Comment:
Localization: American English would prefer `counterclockwise` (https://books.google.com/ngrams/graph?content=counterclockwise,anticlockwise&year_start=1900&year_end=2000&corpus=17&smoothing=3&share=) while British speakers are indifferent - I hear Unity docs prefer US standardization?

Reply:
Ack. Didn't even clock that (geddit?) Thanks for pointing it out!

Reply:
TIL the colonizers use the word "anticlockwise"